.github/workflows/deploy.yml

```diff
@@ -62,12 +62,9 @@ jobs:
 
       - name: Build source and wheel distributions
         run: |
-          # Change the versioneer format to "pre" so that the commit hash isn't
-          # included (PyPI doesn't allow it). Can't do this permanently because
-          # we rely the hash to indicate to the tests that this is a local
-          # verison instead of a published version.
-          sed --in-place "s/pep440/pep440-pre/g" setup.cfg
-          python setup.py sdist bdist_wheel
+          # Use the no-local-version configuration on setuptools-scm because
+          # is the only one compatible with Test PyPI.
+          HARMONICA_VERSION_LOCAL_SCHEME="no-local-version" python setup.py sdist bdist_wheel
```

Comment:
Instead of using an env variable, it's better to apply the same hack Leo was doing, mainly because once we move away from `setup.py` to `pyproject.toml` we won't have the option to check if an env variable exists or not.

Reply:
We are going to set `local_scheme = "no-local-version"` by default, so no shananginan is required.